client/src/shared/components/Table/index.tsx

```diff
@@ -44,15 +44,17 @@ const Table: React.FC<TableProps> = ({
   };
 
   return (
-    <div className="overflow-auto text-neutral-900 flex justify-center rounded-[5px]">
-      <table className="text-left w-full">
-        <thead className="bg-neutral-200">
+    <div className="grid">
+      <table className="rounded-t-lg overflow-hidden">
+        <thead>
           <tr>
-            <th className="p-4">
-              {checkbox && <input type="checkbox" className="h-5 w-5 hover:bg-sky-700" />}
-            </th>
+            {checkbox && (
+              <th className="bg-lightGray2 h-[49px]">
+                <input type="checkbox" className="h-5 w-5 hover:bg-sky-700" />
+              </th>
+            )}
```

Comment:
This causes a broken design for tables with checkboxes on rows but none on columns. (see image)
![image](https://github.com/framgia/sph-lms/assets/116238730/043ac302-1777-465d-9b94-2476278295e1)


```suggestion
            {selectableRows && (
              <th className="bg-lightGray2 h-[49px]">
              {checkbox && <input type="checkbox" className="h-5 w-5 hover:bg-sky-700" />}
              </th>
          )}
```

add `selectableRows?: boolean` to TableProps and pass default into Table: `selectableRows = true`